doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -2,9 +2,13 @@
 
 # Log Persistence
 
-Logs (both system and application logs) are useful for troubleshooting Ray applications and Clusters. For example, you may want to access system logs if a node terminates unexpectedly.
+Logs (both system and application logs) are useful for troubleshooting Ray applications and clusters.
+For example, if a node terminates unexpectedly, you can check the logs to see what happened.
 
-Similar to Kubernetes, Ray does not provide a native storage solution for log data. Users need to manage the lifecycle of the logs by themselves. This page provides instructions on how to collect logs from Ray Clusters that are running on Kubernetes.
+Ray does not provide a default solution for log data persistence.
+If a job runs into completion, the logs will be lost.
```

Comment:
If a `RayJob` is run, by default the RayCluster stays up unless `shutdownAfterJobFinishes` is set, so the logs are actually still available in this case.